### PR TITLE
add `listServiceSpecificCredentials`, `deleteServiceSpecificCredential` to IAM

### PR DESCRIPTION
**Pull request by nealio82**
* Add `listServiceSpecificCredentials` and `deleteServiceSpecificCredential` to IAM client
* Fill in tests
* Add documentation

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -247,9 +247,11 @@
                 "CreateServiceSpecificCredential",
                 "CreateUser",
                 "DeleteAccessKey",
+                "DeleteServiceSpecificCredential",
```
Comment (jderusse) [inline manifest.json:250]: Don't forget the changelog ;)

Comment (nealio82) [reply manifest.json:250]: Thanks for the reminder @jderusse - updated :)

File: src/Service/Iam/CHANGELOG.md [hunk 2]
```diff
@@ -2,13 +2,20 @@
...
+- Added `IamClient::listServiceSpecificCredentials()`
+- Added `IamClient::deleteServiceSpecificCredential()`
+
 ## 1.3.0
 
 ### Added
 
 - Added `IamClient::putUserPolicy()`
 - Added `IamClient::deleteUserPolicy()`
-- Added `IamClient::::createServiceSpecificCredential()`
```
Comment (nealio82) [inline src/Service/Iam/CHANGELOG.md:11]: Spotted / fixed this while I was here

**Review by jderusse — APPROVED** [review]
I've updated php-cs-fixer, you'll need to `composer update` + `generate --all` to fix the CI

**Comment by jderusse** [comment]
thank you

**Pull request merged**